llm_demo/orchestrator/langchain_tools/tools.py

```diff
@@ -223,6 +223,38 @@ async def list_tickets():
     return list_tickets
 
 
+def generate_list_seats(client: aiohttp.ClientSession):
```

Comment:
Might wanna add a `class SeatInput(BaseModel)` (can refer to the other input class at the top).

Reply:
I'm also wondering if we want to add `seat_row` and `seat_letter` as optional input in the `insert_ticket` tools above. (I'm assuming we will allow user to insert ticket without picking a seat? Or do they have to pick a seat?

If change, also need to update the examples for this tool below.

Reply:
Great idea, done! And adding seat_row and seat_letter to insert_ticket as well in the next commit just shortly.